Library/ContextStorageAdapters/ElasticsearchAdapter.php

```diff
@@ -177,7 +225,7 @@ private function initCommonRequestParams()
      *
      * @throws ContextStorageException
      */
-    private function initClient()
+    private function _initClient()
```

Comment:
do we really want to follow this convention ?

Reply:
just forgot to remove this one

Reply:
no !